dockstore-cli-reports/generated/src/main/resources/pom.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~    Copyright 2017 OICR
```

Comment:
Copyright year and UCSC

Reply:
Interesting, I never read these because they're generated. Turns out, whatever plugin we're using copies the copyright notice from the module pom